README.md

```diff
@@ -1,51 +1,44 @@
 # The impact of undesired technical variability on large-scale data compendia
 
-**Motivation:**  
-In the last two decades, scientists working in different labs have assayed gene expression from millions of samples. These experiments can be combined into a compendium in order to gain a systematic understanding of biological processes. However, combining different experiments introduces technical variance, which could distort biological signals in the data leading to misinterpretation. 
+**Alexandra Lee and Casey Greene 2020**
```

Comment:
Are you putting manuscript authors' names on here or did you want to keep it just you and Casey?

Reply:
Good point.  I'll add the other authors to match the impending manuscript.